aws-iam-instance-profile/README.md

```diff
@@ -39,7 +39,8 @@ resource "aws_instance" "instance" {
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:-----:|
-| enable\_ssm | Attach the appropriate policies to allow the instance to integrate with AWS Systems Manager. | `string` | `"true"` | no |
+| create\_role | Creates a role for use with the instance profile. | `bool` | `true` | no |
```

Comment:
Instead of having a a second parameter, can we use the fact that `exiting_role_name` is null as a signal to create a new role?